deb/build/debian/control

```diff
@@ -2,7 +2,7 @@ Source: @@ARTIFACTNAME@@
 Section: devel
 Priority: extra
 Maintainer: @@AUTHOR@@
-Build-Depends: debhelper (>= 9)
+Build-Depends: debhelper (>= 9), dh-systemd
```

Comment:
You can use the Oracle JDK 8 on that version without issue.
(hence partly why the init script has such a convoluted way of finding which java to use)